pil-analyzer/src/condenser.rs

```diff
@@ -409,6 +491,46 @@ impl<'a, T: FieldElement> Condenser<'a, T> {
     }
 }
 
+impl<'a, T: FieldElement> From<IdentityWithoutID<AlgebraicExpression<T>>> for Value<'a, T> {
+    fn from(id: IdentityWithoutID<AlgebraicExpression<T>>) -> Self {
+        match id.kind {
+            IdentityKind::Polynomial => {
+                let id = id.into_identity(0);
+                let (left, right) = id.as_polynomial_identity();
+                let right = right
+                    .cloned()
+                    .unwrap_or_else(|| AlgebraicExpression::Number(0.into()));
+                let fields = vec![Arc::new(left.clone().into()), Arc::new(right.into())];
+
+                Value::Enum("Identity", Some(fields))
+            }
+            IdentityKind::Plookup | IdentityKind::Permutation => {
+                let name = if id.kind == IdentityKind::Plookup {
+                    "Lookup"
+                } else {
+                    "Permutation"
```

Comment:
Why do we need these as Strings?

Reply:
Enum variants in pil are identified by their name. I don't think it makes sense to introduce identifiers, it makes it really hard to debug and probably not too much faster.